Scripts/Modules/MultipleRayInputModule/MultipleRayInputModule.cs

```diff
@@ -1,3 +1,6 @@
+#if !UNITY_EDITORVR
+#pragma warning disable 67, 414, 649
```

Comment:
Rather than disabling these for the whole file, we should wrap preProcessRaycastSource in an #if UNITY_EDITORVR.

Or it might make sense for this class to be public again, since it's actually pretty useful as a stand-alone module.